Source/IGListCollectionViewLayout.h

```diff
@@ -128,6 +128,21 @@ NS_SWIFT_NAME(ListCollectionViewLayout)
                       topContentInset:(CGFloat)topContentInset
                         stretchToEdge:(BOOL)stretchToEdge;
 
+/**
+ Create and return a new vertically scrolling collection view layout.
+ 
+ @param stickyHeaders Set to `YES` to stick section headers to the top of the bounds while scrolling.
+ @param showHeaderWhenEmpty Set to `YES` to show sticky header when a section had no item
+ @param topContentInset The top content inset used to offset the sticky headers. Ignored if stickyHeaders is `NO`.
+ @param stretchToEdge Specifies whether to stretch width of last item to right edge when distance from last item to right edge < epsilon(1)
+ 
+ @return A new collection view layout.
+ */
+- (instancetype)initWithStickyHeaders:(BOOL)stickyHeaders
+                  showHeaderWhenEmpty:(BOOL)showHeaderWhenEmpty
+                      topContentInset:(CGFloat)topContentInset
+                        stretchToEdge:(BOOL)stretchToEdge;
```

Comment:
Could we actually just include this as a mutable `@property BOOL showHeaderWhenEmpty`? Trying to prevent some initializer creep here, ha!

Reply:
Sure, I will roll it back and extract the property `showHeaderWhenEmpty ` to public.